src/scripts/windows/build.cmd

```diff
@@ -58,14 +62,14 @@ goto :init
 :main
 
     set SRC_PATH=%REPO_ROOT_PATH%\src
-    set BUILD_PATH=%REPO_ROOT_PATH%\_build\Windows\%VSCMD_ARG_TGT_ARCH%\%BUILD_TYPE%
+    set BUILD_PATH=%SRC_PATH%\_build\Windows\%BUILD_ARCH%\%BUILD_TYPE%
```

Comment:
this ensures that VS builds use the same cmake cache as command line

Reply:
I don't understand. Before this change, both VS and command line would build into <xlang_root>\_build. I don't see a corresponding change to CMakeSettings.json to target <xlang_root>\src\_build

Reply:
Not my experience.  Command line would build into <xlang_root>_build and VS into <xlang_root>\src\_build

Reply:
(I even tried experimenting with current dir and build.cmd always build into <xlang_root>_build)

Reply:
Apparently, it depends on where you open VS from. I always open it from the xlang root and so it hangs _build off there. I just tried opening VS from /src directory and it created the _build directory there. <sigh>

Since you're already mucking about in here, can we figure out how to configure CmakeSettings.json to ALWAYS build it in the same place (/src/_build seems like the logical place)

Reply:
I'll take a look

Reply:
ok, I can launch devenv root, with current directory pointing to root cmake cache (not under src) to enable Intellisense in VS - will back this out.